libcaf_core/caf/actor_system_config.hpp

```diff
@@ -252,6 +252,13 @@ class actor_system_config {
   size_t work_stealing_relaxed_steal_interval;
   size_t work_stealing_relaxed_sleep_duration_us;
 
+  // -- config parameters for the logger ---------------------------------------
+
+  std::string logger_filename;
+  std::string logger_verbosity;
```

Comment:
A `string` for one of 6 possible options feels quite overkill to me. Why not use an `atom` here? Atoms also clean up the `if-else` trees by allowing `switch` instead.